Source/JavaScriptCore/wasm/js/JSToWasm.cpp

```diff
@@ -25,6 +25,10 @@
 
 #include "config.h"
 #include "JSToWasm.h"
+#include "bmalloc/Gigacage.h"
+#include "wasm/WasmCallee.h"
```

Comment:
`wasm/` is not necessary.

Reply:
@danlliu Do you use clangd? You should pass `-header-insertion=never` to your clangd plugin

Reply:
Yup, I set that up last time but forgot to set it up on my new machine; I've updated that now.